docs/essentials/collections/selectingaCollectionClass.md

```diff
@@ -1 +1,69 @@
+# Selecting a Collection Class
 
+Be sure to choose your collection class carefully. Using the wrong type can restrict your use of the collection. In general, avoid using the types in the System.Collections namespace unless you are specifically targeting .NET Framework version 1.1. The generic and concurrent versions of the collections are to be preferred because of their greater type safety and other improvements.
+
+Consider the following questions:
+
+* Do you need a sequential list where the element is typically discarded after its value is retrieved? 
+
+    * If yes, consider using the [System.Collections.Queue](http://dotnet.github.io/api/System.Collections.Queue.html) class or the [System.Collections.Generic.Queue&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.Queue%601.html) generic class if you need first-in, first-out (FIFO) behavior. Consider using the [System.Collections.Stack](http://dotnet.github.io/api/System.Collections.Stack.html) class or the [System.Collections.Generic.Stack&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.Stack%601.html) generic class if you need last-in, first-out (LIFO) behavior. For safe access from multiple threads, use the concurrent versions [System.Collections.Concurrent.ConcurrentQueue&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentQueue%601.html) and [System.Collections.Concurrent.ConcurrentStack&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentStack%601.html).
```

Comment:
I might not even mention the System.Collections.Stack and System.Collections.Queue (since we recommend not using them)